models/csam-report.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (sequelize, DataTypes) => {
+  const CSAMReport = sequelize.define('CSAMReport', {
+    accountSid: DataTypes.STRING,
+    twilioWorkerId: DataTypes.STRING, // The worker who submited the report
```

Comment:
spelling: submitted

Reply:
The primary risk would be if one of us deleted a worker from the Twilio console or API. We've discussed this concern before and so far kicked the can down the road. We could potentially maintain a Workers table in our own database, which then keeps different parts of that data duplicated across three systems: Okta, Twilio and us. Definitely something to consider as we look at potential database and data model adjustments early in 2022.